### DISCUSSION TITLE
Wolf like support for sunshine

### DISCUSSION THREAD

**Discussion by anthr76** [LizardByte/community#399] [Sunshine]
Hello! I'm looking to accomplish a wolf like experience on a recently built living room gaming build. By default the PC boots into gamescope-session and I'm able to connect to sunshine with it!

![image](https://github.com/LizardByte/Sunshine/assets/25143877/b8ea6c5f-42fe-4884-aec6-fd4369b7e737)

What I'm now trying to accomplish is something similar to wolf where I can boot into a virtual display and desktop environment separate from TTY1's physical display of gamescope session. I also would like to do this with wayland. I've been messing around with https://gitlab.freedesktop.org/ofourdan/xwayland-run specifically `wlheadless-run` with no luck so far. It seems to always bring me back to my physical display. 

I'm curious has anyone done anything like this yet? Any tips to provide

**Comment by ReenigneArcher** [comment]
I've been working on adding a virtual display for the upcoming unit testing PR.

Planning to continue more work on this after the upcoming patch release (v0.22.1). It will also depend on https://github.com/LizardByte/Sunshine/pull/2185

So far my solution is on for xorg, but I will look into doing the same for wayland.

[a comment by an automated account is not shown]
